akka-http-core/src/test/scala/akka/http/scaladsl/model/EntityDrainingSpec.scala

```diff
@@ -0,0 +1,95 @@
+package akka.http.scaladsl.model
```

Comment:
We're in a no blame culture here 😎 
Thanks for adding the missing copy @krzemin :)